packages/cubejs-query-orchestrator/src/orchestrator/RedisPool.ts

```diff
@@ -1,23 +1,22 @@
+import Redis from 'ioredis';
```

Comment:
I think, It's better to import redis directly, instead of using default export.

```suggestion
import { Redis } from 'ioredis';
```